doc/source/array.rst

```diff
@@ -1145,6 +1155,19 @@ Here's a usage example::
 
     my_dot_prod = krnl(a, b).get()
 
+Or by specifying the output::
+
+    from pycuda.curandom import rand as curand
+    a = curand((10, 200), dtype=np.float32)
+    red = ReductionKernel(np.float32, neutral=0,
+                               reduce_expr="a+b",
+                               arguments="float *in")
+    a_sum = gpuarray.empty(10, dtype=np.float32)
```

Comment:
How come this is created as size-10, when only 1 element is used?

Reply:
The example fills each element of a_sum by calling the reduction kernel 10 times.

Reply:
OIC, nvm.